config/settings_data.json

```diff
@@ -15,6 +15,7 @@
       "footer_column2_content": "<ul><li>Content 2</li></ul>",
       "footer_column3_title": "Column 3",
       "footer_column3_content": "<ul><li>Content 3</li></ul>",
+      "pageSizes": [10, 20, 50, 100],
```

Comment:
Why we have pageSizes here now? Where we have any business requirements for this?)

Reply:
It’s a temporary solution for static template(just to see how it will look), we will use our constant with pageSizes after connecting templates with Vue